### PR TITLE
Add task completion endpoint

### PR DESCRIPTION
**Pull request by aeldamaty**
Refs #SDE-3321

### CODE REVIEW

File: source/includes/projects.md [hunk 1]
```diff
@@ -130,21 +130,90 @@ Content-Type: application/json
...
-                "complete": 2,
-                "na": 0
+        "task_statuses": {
+            "development": {
+                "medium": {
+                    "na": 0,
+                    "complete": 0,
+                    "incomplete": 15
+                },
+                "complete": 0,
```
Comment (emintham) [inline source/includes/projects.md]: I'd just remove these "sum" type fields. The project overview page uses it but it only needs to calculate it for one single project and wouldn't be too costly. This keeps the response cleaner and you can just loop over all the key values under a phase and know that you'll get priority as keys.

**Comment by emintham** [comment]
+1

**Comment by emintham** [comment]
@aeldamaty You'd need to assign a milestone I think.

**Comment by harry-kim** [comment]
+1

**Pull request merged**